p2p/p2p.proto

```diff
@@ -50,6 +50,12 @@ message GetDocumentRequest {
   AccessType access_type = 2;
   bytes nft_registry_address = 3;
   bytes nft_token_id = 4;
+  AccessTokenRequest access_token_request = 5;
```

Comment:
why did you create a separate message here instead of just adding the two fields? It seems to add a lot complexity in the implementation for not much gain.

Reply:
I agree that is 1 field vs 2 fields extra. From the client perspective is easier to see if the access_token_request is provided or not as a whole (accessTokenRrequest != nil). In addition, semantically have the token as a block makes sense.